apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleSection.vue

```diff
@@ -1,17 +1,15 @@
 <template>
     <p-pane-layout>
         <section-header :title="$t('INVENTORY.COLLECTOR.DETAIL.SCHEDULE')"
-                        :edit-mode="state.isEditMode"
+                        :edit-mode="state.isEditMode || !collectorFormState.schedulePower"
```

Comment:
suggestion: I think this logic is pretty duplicated with the watcher in the bottom of this file.